wfl/fit/error.py

```diff
@@ -216,7 +216,7 @@ def _reshape_normalize(quant, prop, atoms, per_atom):
         for cat in all_diffs[prop]:
             diffs = np.asarray(all_diffs[prop][cat])
             weights = np.asarray(all_weights[prop][cat])
-
+         
```

Comment:
Please clean up changes that are just whitespace.

Reply:
I don't understand why this is counted as 'diff' here. I don't see any difference between this script and one from the main. It has the same white space.

Reply:
tab vs. spaces?

Reply:
I hope this problem is resolved in current script.